plugins/woocommerce/tests/php/includes/class-wc-products-tracking-test.php

```diff
@@ -33,6 +33,49 @@ public function tearDown(): void {
 	 */
 	public function test_product_add_publish(): void {
 		$product = new WC_Product_Simple();
+		$product->set_name( 'New name' );
```

Comment:
I added the `name` as otherwise when you trigger `$product->save();` it will deem it as a brand new project every time. Adding a `name` causes it to update the same product.